.github/PROCESS.md

```diff
@@ -57,6 +57,13 @@ We also look at the flows' metrics to gather more information about the team's d
 Estimation mistakes should be an exception, not a norm.
 Try to estimate on the planning as best as you can.
 
+## Daily progress
+
+We want our work to be visible and transparent.
+This way we can help each other and make sure that we are on the right track.
+To uphold these principles, we agree to demonstrate our progress daily through various means such as
```

Comment:
I think "demonstrate" sends the wrong message. We want daily progress to be present and (typically) public, but there is no need to show it off just for demonstration or, worse, write daily reports or something. (Of course, if someone _wants_ to demonstrate it, that's fine, but that's not the goal).

Reply:
Please send the exact suggestion that will satisfy your needs. As far as I understand, you don't want daily progress to only exist but also to be visible.